content/axon-server-installation-guides/modules/ROOT/pages/installing_as_docker-k8s.adoc

```diff
@@ -0,0 +1,604 @@
+The mapping for /eventdata must be changed to /axonserver/events, and the mapping for /data must be changed to /axonserver/data.
+
+____
+
+== Building you own image [[build-image]]
+
+A starter Dockerfile is included below which can be tailored as per your requirements.
+
+The starter file helps create the image in multiple stages,
+
+* The image will be based on a compact image from Google’s “distroless” base images at the gcr.io repository, in this case “gcr.io/distroless/java:11”.
+* The first stage creates the directories that will become our volumes.
+This step cannot be performed in the Distroless image, because that image does not provide a shell.
+* The second stage begins by copying the home directory with its volume mount points, carefully keeping ownership set to the new user.
+* The last steps copy the executable jar named axonserver.jar and a common set of properties.
+It marks the volume mounting points and exposed ports and finally specifies the command to start Axon Server EE.
+
+[source,bash]
+----
+FROM busybox as source
+
+RUN mkdir -p /axonserver/config /axonserver/data /axonserver/events /axonserver/log /axonserver/exts
+
+FROM gcr.io/distroless/java:11
+
+COPY --from=source /axonserver /axonserver
+COPY axonserver.jar axonserver.properties /axonserver/
+
+WORKDIR /axonserver
+
+VOLUME [ "/axonserver/config", "/axonserver/data", "/axonserver/events", "/axonserver/log", "/axonserver/exts", "/axonserver/plugins"  ]
+EXPOSE 8024/tcp 8124/tcp 8224/tcp
+
+ENTRYPOINT [ "java", "-jar", "./axonserver.jar" ]
+
+----
+
+If you want to build a "nonroot" version, you need to adjust this as follows:
+
+[source,text]
+----
+FROM busybox as source
+RUN addgroup -S -g 1001 axonserver \
+    && adduser -S -u 1001 -G axonserver -h /axonserver -D axonserver \
+    && mkdir -p /axonserver/config /axonserver/data /axonserver/events /axonserver/log /axonserver/exts \
+    && chown -R axonserver:axonserver /axonserver
+
+FROM gcr.io/distroless/java:11
+
+COPY --from=source /etc/passwd /etc/group /etc/
+COPY --from=source --chown=axonserver /axonserver /axonserver
+
+COPY --chown=axonserver axonserver.jar axonserver.properties /axonserver/
+
+USER axonserver
+WORKDIR /axonserver
+
+VOLUME [ "/axonserver/config", "/axonserver/data", "/axonserver/events", "/axonserver/log", "/axonserver/exts", "/axonserver/plugins" ]
+EXPOSE 8024/tcp 8124/tcp 8224/tcp
+
+ENTRYPOINT [ "java", "-jar", "./axonserver.jar" ]
+
+----
+
+As you can see this will start by creating the user "`axonserver`" belonging to a group with the same name.
+When copying the directory, we now have to ensure that ownership transfers correctly and specify the user to run as, but otherwise it looks pretty similar.
+
+For the common properties (axonserver.properties), the minimum set can be added to ensure that the volumes get mounted and logs generated.
+Again these can be tailored as per the deployment requirements.
+
+[source,text]
```

Comment:
Better set to [source, properties] (there is a syntax highlighter for .properties)